manager/controllers/baseline_update.go

```diff
@@ -86,6 +88,7 @@ func BaselineUpdateHandler(c *gin.Context) {
 		LogAndRespError(c, err, "Database error")
 		return
 	}
+	kafka.EvaluateBaselineSystems(&baselineID, account)
```

Comment:
During update you need to reevaluate both old systems removed from baseline and new systems added to the baseline.
This will IMHO update only new sytems.